### DISCUSSION TITLE
Upgrade from v2 to v3

### DISCUSSION THREAD

**Discussion by fdecourt** [lukevella/rallly#872] [Q&A]
Hi everybody,
I tried to upgrade from v2 to v3, but it was unsuccessfull, saying that the posgreSQL v15 schema is not compatible with  the v16 version.
I am using docker and watchtower to do the upgrade (so I temporaly fixed the version to v2.1.11 and v15.4 for posgreSQL).
Any tips on how to upgrade without loosing data ?
As a non tech guy, I use the MariusHosting script : [https://mariushosting.com/how-to-install-rallly-on-your-synology-nas/]
Thanks for your help !

**Comment by lukevella** [comment]
Rallly currently runs on Postgres v14 and I haven't tested it on newer versions of Postgres. It sounds like it won't run on Postgres v16 so you may need to downgrade Postgres to continue using it.

**Comment by fdecourt** [comment]
Hi Luke,
Thanks for your answer. PosgreSQL was updated to v16 automatically. I will try to use the v3 on the actual v15.4 I am using (it is working well with the v2).
Regards,
F.

**Reply by fdecourt** [reply]
Ok, so I installed the v3 on the v15.4 PosgreSQL, and it seems to be working well. Thanks for your help.
Regards,
